modules/osdk-managing-psa-for-operators-with-escalated-permissions.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-complying-with-psa.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-managing-psa-for-operators-with-escalated-permissions.adoc_{context}"]
+= Managing pod security admission for Operator workloads that require escalated permissions
+
+If your Operator project requires escalated permissions to run, you must edit your Operator's cluster service version (CSV) similar to the following examples.
+
+.Procedure
+
+. Set the security context configuration to the required permission level in your Operator's CSV, similar to the following example:
++
+.Example `<operator_name>.clusterserviceversion.yaml` file with network administrator privileges
+[source,yaml]
+----
+containers:
+   - name: my-container
+     securityContext:
+       allowPrivilegeEscalation: false
+       capabilities:
+         add:
+           - "NET_ADMIN"
+----
+
+. Set the service account privileges that allow your Operator's workloads to use the required security context constraints (SCC), similar to the following example:
++
+.Example `<operator_name>.clusterserviceversion.yaml` file
+[source,yaml]
+----
+ install:
```

Comment:
This line is indented one space, is that intentional or can it be shifted left? (might need to check the rest as well)